src/dao/MoodDao.java

```diff
@@ -31,7 +41,41 @@ public MoodDao() {
      * executing the query
      */
     public ArrayList<Mood> getAllMoods() throws DaoException {
-        return null;
+        BufferedReader reader = null;
+        try {
+            ArrayList<Mood> moods = new ArrayList<Mood>();
+            String ln = "";
+
+            reader = new BufferedReader(new FileReader("data/Moods.csv"));
+            reader.readLine();
+
+            while ((ln = reader.readLine()) != null) {
+                String[] obj = ln.split(COMMA_DELIMITER);
+                if (obj.length > 0) {
```

Comment:
It would be better to check that the array length matches the number of expected fields to avoid ArrayIndexOutOfBounds exceptions.